### ISSUE TITLE
Incomplete line in NHC dat track file cannot be read by ASGS

### ISSUE THREAD

**Issue by carolakaiser** [StormSurgeLive/asgs#904]
When running ASGS with an original NHC DAT track file that is misconfigured/incomplete, ASGS has issues to interpret this line correctly.

Original file:
AL, 09, 2004091618,   , BEST,   0, 325N,  874W,  50,  975, TS,  50, NEQ,   20,   20,    0,    0,    0,    0,   0,  75,   0, 
AL, 09, 2004091700,   , BEST,   0, 338N,  865W,  30,  986, TD,   0,    ,    0,    0,    0,    0,    0,    0,   0,  40,   0, 
AL, 09, 2004091706,   , BEST,   0, 347N,  857W,  25,  991, TD,   0,    ,    0,    0,    0,    0, 
AL, 09, 2004091712,   , BEST,   0, 354N,  840W,  20,  994, TD,   0,    ,    0,    0,    0,    0, 
AL, 09, 2004091718,   , BEST,   0, 362N,  823W,  20,  996, TD,   0,    ,    0,    0,    0,    0, 
AL, 09, 2004091800,   , BEST,   0, 370N,  805W,  20,  999, TD,   0,    ,    0,    0,    0,    0, 
AL, 09, 2004091806,   , BEST,   0, 377N,  785W,  15,  998, TD,   0,    ,    0,    0,    0,    0, 
AL, 09, 2004091812,   , BEST,   0, 384N,  767W,  15, 1000, TD,   0,    ,    0,    0,    0,    0, 
AL, 09, 2004091818,   , BEST,   0, 380N,  755W,  25, 1002, EX,   0,    ,    0,    0,    0,    0, 
**AL, 09, 2004091900,   , BEST,   0, 375N,  740W,  35, 1003, EX,  34, NEQ,    0,    0,    0,   60,** 

The last line cannot be read and causes the run to stop at an earlier time (2004091818).

There are two possible workarounds that have been tested and that work:
1: Use a complete line for the last time like
AL, 09, 2004091900,   , BEST,   0, 325N,  874W,  50,  975, TS,  **50, NEQ,   20,   20,    0,    0,    0,    0,   0,  75,   0,** 

2: Set the column 'NEQ' to empty
AL, 09, 2004091900,   , BEST,   0, 375N,  740W,  35, 1003, EX,  **34,    ,    0,    0,    0,    0,    0,    0,   0,  60,   0,** 

It has not been tested if such an incomplete line would lead to a stop of ASGS if that line was in the middle of the file or if it just would be skipped.

Thank you for your consideration to address this issue.

**Comment by wwlwpd** [comment]
@jasonfleming and I have kind of discussed this because of some resiliency that would be nice to add due to the replay server test; but I can see how this would be beneficial during both operations and for hindcast/experiments.

I have found that this is affected by `COLDSTARTDATE` and `HINDCASTLENGTH`, so maybe there is an issue in this calculation in the scripts (or done by hand). This is how I fixed an issue with the replay server that was missing the earliest record that was being provided after the _nowification_. So there are some options here, but if the intent is really to add a ghost record, then the determination of the values in that record is going to require domain expertise I do not have. That all said, I defer to @jasonfleming.
